cmds/task/fetchers.go

```diff
@@ -58,7 +58,45 @@ func runStatus(credentials *tcclient.Credentials, args []string, out io.Writer,
 	fmt.Fprintln(out, getRunStatusString(s.Status.Runs[runID].State, s.Status.Runs[runID].ReasonResolved))
 	return nil
 }
+// confirmMsg displays confirmation message when --confirm is used
+func confirmMsg(command string, credentials *tcclient.Credentials, args []string, out io.Writer, _ *pflag.FlagSet )(string) {
```

Comment:
No need to include unused parameters here - `out` and the FlagSet are unused.  I'm not sure why the linter didn't catch that..

Also, it would be preferable to pass the taskId directly, rather than passing args.  Then this function doesn't have to know about the structure of the args.  Everywhere this function is called already has the taskId.